utilities/SimpleIngest.js

```diff
@@ -48,16 +57,27 @@ class SimpleIngest extends Utility {
     const libInfo = await this.concerns.ArgLibraryId.libInfo();
 
     const type = R.path(["metadata", "abr", "mez_content_type"], libInfo);
-    if(R.isNil(type)) throw Error("Library does not specify content type for simple ingests");
+    if (R.isNil(type))
+      throw Error("Library does not specify content type for simple ingests");
 
-    const libABRProfile = R.path(["metadata", "abr", "default_profile"], libInfo);
-    if(R.isNil(libABRProfile)) throw Error("Library does not specify ABR profile for simple ingests");
+    const libABRProfile = R.path(
+      ["metadata", "abr", "default_profile"],
+      libInfo
+    );
+    if (R.isNil(libABRProfile))
+      throw Error("Library does not specify ABR profile for simple ingests");
 
-    const libMezManageGroups = R.path(["metadata", "abr", "mez_manage_groups"], libInfo);
+    const libMezManageGroups = R.path(
+      ["metadata", "abr", "mez_manage_groups"],
+      libInfo
+    );
 
-    const libMezPermission = R.path(["metadata", "abr", "mez_permission_level"], libInfo);
+    const libMezPermission = R.path(
+      ["metadata", "abr", "mez_permission_level"],
+      libInfo
+    );
 
-    const {drm, libraryId, title} = this.args;
+    const { drm, libraryId, title, configUrl, description } = this.args;
     const encrypt = true;
```

Comment:
encrypt should be false for NFTs?

Reply:
Latest makes encrypt a command line param with default being "false"